### PR TITLE
wip: next

### PR DESCRIPTION
**Pull request by tmm**
Intent to release the next version of wagmi.

Your ENS/address: awkweb.eth

Closes https://github.com/tmm/wagmi/issues/257 https://github.com/tmm/wagmi/issues/307 https://github.com/tmm/wagmi/issues/354

### CODE REVIEW

**Comment by albertputrapurnama** [comment]
@tmm are there any action items list for releasing 0.3.0?

**Comment by bigint** [comment]
wen release 👻 ?

**Comment by tmm** [comment]
Can publish another beta release tomorrow. There's a few more things to tie off before the final release (like #328). It will be worth it!

**Comment by bigint** [comment]
@tmm Will this be handled? when rejecting the signed data it's throwing an unhandled error, but rejecting the write action works fine 🤔 

https://user-images.githubusercontent.com/69431456/162684692-2db43629-af8e-45fd-880b-815f08345135.mov

**Comment by outdoteth** [comment]
ETA?

**Comment by tmm** [comment]
@outdoteth soon! In the meantime, you can use [`wagmi@0.3.0-next.19`](https://www.npmjs.com/package/wagmi/v/0.3.0-next.19) and the new docs at https://next.wagmi.sh.

File: docs/pages/index.en-US.mdx [hunk 1]
```diff
@@ -86,10 +86,3 @@ Help support future development and make wagmi a sustainable open-source project
 - [awkweb.eth](https://etherscan.io/enslookup-search?search=awkweb.eth)
 - [GitHub Sponsors](https://github.com/sponsors/tmm?metadata_campaign=docs_support)
 - [Gitcoin Grant](https://gitcoin.co/grants/4493/wagmi-react-hooks-library-for-ethereum)
-
-## Authors
-
-- awkweb.eth ([@awkweb](https://twitter.com/awkweb)) – [Mirror](https://mirror.xyz)
-- moxey.eth ([@jakemoxey](https://twitter.com/jakemoxey)) – [Rainbow](https://rainbow.me)
-
-Thanks to julianhutton.eth ([@julianjhutton](https://twitter.com/julianjhutton)) for providing the awesome logo!
```
Comment (tmm) [inline docs/pages/index.en-US.mdx]: put in readme instead

**Pull request merged**